docs/user/tutorials/01-10-add-delete-modules.md

```diff
@@ -0,0 +1,29 @@
+# Adding and Deleting a Kyma Module Using Kyma CLI
+This tutorial shows how you can add a new module using Kyma CLI. 
+
+> [!WARNING]
+> Modules added without any specified form of the custom resource have the policy field set to `Ignore`.
+## Procedure
+
+### Adding a New Module
+
+1. To add a new module with the default policy set to `CreateAndDelete`, use the following command:
```

Comment:
```suggestion
To add a new module with the default policy set to `CreateAndDelete`, use the following command:
```
There are only a few different commands that do almost the same thing, and not subsequent steps, so I think we can omit numbering.